### ISSUE TITLE
reduce number of messages sent by grouping them

### ISSUE THREAD

**Issue by fkleedorfer** [researchstudio-sat/webofneeds#731]
After #729, mails are only sent for events that happen when a user is not logged in.

However, we still send one mail per message received, which may be a bit much.

We should implement some kind of grouping, ideally configurable by the user.

requirements are:
- be informed quickly of important events
- do not allow users to cause too many mails to be sent to other users
- do not get too many messages in total
- do not get too large messages
- do not

**Comment by fkleedorfer** [comment]
Cleaning up old issues. Out of scope for now
